js/js.js

```diff
@@ -1,3 +1,28 @@
-$(function () {
-  $('[data-toggle="tooltip"]').tooltip()
-})
\ No newline at end of file
+$(document).ready(function() {
+
+  $(function () {
+    $('[data-toggle="tooltip"]').tooltip()
+  });
+
+  // open modal window if a hash is found in URL
+  if(window.location.hash) {
+    $(window.location.hash).modal('show');
+    // add direct link to modal window
+    $(".link").empty();
+    $(".link").append("<small>Link to this command: "+window.location.href+"</small>");
```

Comment:
Possibly:
`$(".link").append("<small>Link to this command: <a href=&quot;"+window.location.href+"&quot;>"+window.location.href+"</a></small>");`